types/validation.d.ts

```diff
@@ -2,32 +2,32 @@ declare module 'mongoose' {
 
   type SchemaValidator<T> = RegExp | [RegExp, string] | Function | [Function, string] | ValidateOpts<T> | ValidateOpts<T>[];
```

Comment:
Without changing `SchemaValidator`, and `SchemaTypeOptions`, there's no way for user to set `M`?